code/modules/vore/resizing/resize_vr.dm

```diff
@@ -180,82 +180,94 @@ var/const/RESIZE_A_SMALLTINY = (RESIZE_SMALL + RESIZE_TINY) / 2
-			var/calculated_damage = damage/2 //This will sting, but not kill. Does .5 to 2.625 damage, randomly, to each limb.
+			now_pushing = 0
+			tmob.Weaken(4)
+			var/size_damage_multiplier = 0.25 * (src.size_multiplier / tmob.size_multiplier)
+			var/damage = (rand(1,3)* size_damage_multiplier) //Rand 1-3 multiplied by 1 min or 2 max. 1 min 6 max damage to each limb.
+			var/calculated_damage = damage/2 //This will sting, but not kill. Does .5 to 3 damage, randomly, to each limb.
 
 			if(src.m_intent == "run")
 				var/mob/living/carbon/human/H = src
 				if(istype(H) && istype(H.tail_style, /datum/sprite_accessory/tail/taur/naga))
-					src << "Your heavy tail carelessly slides past [tmob],  crushing them!"
-					tmob << "[src] quickly goes over your body, carelessly crushing you with their heavy tail!"
+					src << "Your heavy tail carelessly slides past [tmob], crushing them!"
+					tmob << "<span class='warning'>[src] quickly goes over your body, carelessly crushing you with their heavy tail!</span>"
 					if(istype(tmob,/mob/living/carbon/human))
 						var/mob/living/carbon/human/M = tmob
 						M.drip(0.1)
-						for(var/obj/item/organ/I in M.organs)
+						for(var/obj/item/organ/I in M.organs) // ???
 							tmob.take_overall_damage(calculated_damage, 0) //Due to the fact that this deals damage across random body parts, this should heal quite fast.
 						admin_attack_log(src, M, "trampled [tmob.name] under foot for [damage * 10] damage.", "Was crushed under foot by [H.name] for [damage * 10] damage.", "Crushed [M.name] for [damage * 10] damage.")
 				else
 					src << "You carelessly step down onto [tmob], crushing them!!"
-					tmob << "[src] steps carelessly on your body, crushing you!"
+					tmob << "<span class='warning'>[src] steps carelessly on your body, crushing you!</span>"
 					if(istype(tmob,/mob/living/carbon/human))
 						var/mob/living/carbon/human/M = tmob
 						for(var/obj/item/organ/I in M.organs)
-							tmob.take_overall_damage(calculated_damage, 0) // 5 damage min, 26.25 damage max, depending on size & RNG. If they're only stepped on once, the damage will heal over time.
+							tmob.take_overall_damage(calculated_damage, 0) // 5 damage min, 30 damage max, depending on size & RNG. If they're only stepped on once, the damage will heal over time.
 						M.drip(0.1)
 						admin_attack_log(src, M, "Crushed [tmob.name] under foot for [damage * 10] damage.", "Was crushed under foot by [H.name] for [damage * 10] damage.", "Crushed [M.name] for [damage * 10] damage.")
 				return 1
 
-			if(src.m_intent == "walk") //Oh my.
-				damage = calculated_damage * 3.5 //Multiplies the above damage by 3.5. This means a min of 1.75 damage, or a max of 9.1875. damage to each limb, depending on size and RNG.
+			else //Oh my.
+				damage = calculated_damage * 3 //Multiplies the above damage by 3. This means a min of 1.5 damage, or a max of 9. damage to each limb, depending on size and RNG.
```

Comment:
Minor damage decrease.

Reply:
Use the calculated average for comparison, the difference is negligible.